src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -970,6 +983,11 @@ private async Task ReceiveLoop(ConnectionState connectionState)
             }
             finally
             {
+                if (invocationMessageChannel != null)
```

Comment:
invocationMessageChannel cannot be null

Reply:
Oh copy pasta, can't be null as currently written.